### ISSUE TITLE
Make the styles inline with the component

### ISSUE THREAD

**Issue by dwilt** [chenglou/react-spinner#17]
One of the beautiful things about the React ecosystem is how well it plays with things like Webpack. It would be great if I could just import the component and have the styles inlined for me.

**Comment by epeli** [comment]
Yup. But there's no need to port to inline styles. The component could just inject a style-tag on demand with https://github.com/substack/insert-css for example.

**Comment by tobia** [comment]
Shouldn't this component import its own styles?

    import './react-spinner.css'

Anyways, for anyone having trouble, just import the styles alongside the module:

    import Spinner from 'react-spinner';
    import 'react-spinner/react-spinner.css';
